com.unity.render-pipelines.universal/Editor/ShaderPreprocessor.cs

```diff
@@ -68,9 +68,7 @@ internal class ShaderPreprocessor : IPreprocessShaders
         ShaderKeyword m_ReflectionProbeBlending = new ShaderKeyword(ShaderKeywordStrings.ReflectionProbeBlending);
         ShaderKeyword m_ReflectionProbeBoxProjection = new ShaderKeyword(ShaderKeywordStrings.ReflectionProbeBoxProjection);
         ShaderKeyword m_DeferredLightShadows = new ShaderKeyword(ShaderKeywordStrings._DEFERRED_LIGHT_SHADOWS);
-        ShaderKeyword m_DeferredSoftShadows = new ShaderKeyword(ShaderKeywordStrings._DEFERRED_SHADOWS_SOFT);
```

Comment:
The keywords are being unified between deferred and forward.
https://github.com/Unity-Technologies/Graphics/pull/4535

Reply:
Hey @kaychang-unity, we should really just delete all as done in this PR rather than unified together.

Reply:
@nigeljw-unity are you suggesting #4535 waits for yours to land, so I can delete _DEFERRED_SHADOWS_SOFT. Or should I land #4535 before yours, then you can remove the extra code on your side?

Reply:
I'm ok with either, but I think I would prefer this to land first to minimise code churn, and also to have this as a low complexity test for removing variants in favour of branches, since we can potentially regress on low tier platforms. This has already gone through performance testing and showed a huge build time improvement

Reply:
Ok, I'll wait your PR then.